operator-manifests/mysql/mysql-operator.yaml

```diff
@@ -231,8 +231,8 @@ spec:
       serviceAccountName: mysql-operator
       containers:
       - name: mysql-operator-controller
-        imagePullPolicy: IfNotPresent
         image: iad.ocir.io/oracle/mysql-operator:0.3.0
+        imagePullPolicy: Always
```

Comment:
if is set with a version tag we dont need to pull always, only apply for `latest` tags